docker/release/config/opensearch-dashboards/opensearch-dashboards-docker-entrypoint.sh

```diff
@@ -154,6 +154,16 @@ opensearch_dashboards_vars=(
     telemetry.optIn
     telemetry.optInStatusUrl
     telemetry.sendUsageFrom
+    data_source.enabled
+    data_source.encryption.wrappingKeyName
+    data_source.encryption.wrappingKeyNamespace
+    data_source.encryption.wrappingKey
+    data_source.audit.enabled
+    data_source.audit.appender.kind
+    data_source.audit.appender.path
+    data_source.audit.appender.layout.kind
+    data_source.audit.appender.layout.highlight
+    data_source.audit.appender.layout.pattern
```

Comment:
Hi @kristenTian ,
Are these values applicable to all opensearch dashboard versions? 1.x, 2.x and 3.x?

Reply:
should be 2.x and eventually 2.4

Reply:
In that case, @opensearch-project/engineering-effectiveness  I believe this raises the question about branching again. 1.x is using the our `main` branch to build as well. So 1.3.7 docker will contain this changes as well which I believe can break things.
cc: @dblock

Reply:
We need to move fast with the official image process then.
This change should be put into https://github.com/opensearch-project/dashboards-docker-images as well.

@gaiksaya @rishabh6788 I am ok to merge this for now as 1.x should just ignore these params.

Let me know what you think. Thanks.

Reply:
It should ignore but if the customer uses this values (since they are available), that wont work or break things right? @kristenTian Please confirm

Reply:
Yes, I confirm it won't cause regression.

Reply:
Since @kristenTian confirmed we will merge this for now.

Thanks.